setup.py

```diff
@@ -1,47 +1,48 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from setuptools import setup
+from setuptools import find_packages, setup
+
 from djangocms_picture import __version__
 
+
+REQUIREMENTS = [
+    'django-cms>=3.3.1',
```

Comment:
@czpython let me know if its okay to merge like that, can than continue adaptions in part 2 :)